be/src/formats/parquet/stored_column_reader.cpp

```diff
@@ -246,7 +246,6 @@ Status RepeatedStoredColumnReader::do_read_records(size_t* num_records, ColumnCo
         _delimit_rows(&records_to_read, &num_parsed_levels);
 
         // decode value read from reader
-        // TODO(zc): optimized here
         {
             // ensure enough capacity
             _is_nulls.resize(num_parsed_levels);
```

Comment:
you can use raw_value::make_room to optimize so that memset will be eliminated.

Reply:
Of course that will be better, but this is not the key issue, StoredColumnReadValuesTime2 is the time cost of this part, it's not in the same magnitude with other parts.
And I will do it.

Reply:
I have tested this part, _is_nulls usually keep in same size, resize is very light, while make_room need to allocate memory, so the original code is more effective.